app/models/support_user_token.rb

```diff
@@ -0,0 +1,3 @@
+class SupportUserToken < Token
```

Comment:
I think we don't need STI, but use polymorphic associations:

```rb
class AuthenticationToken
 belongs_to :authenticable, polymorpic: true
end

# and then

support_user = AuthenticationToken.find_by(token: token).authenticable
```

Reply:
👍

Reply:
I'm struggling a bit to make this work, could you either refine your example or give me a shout when you have a minute to pair?

Reply:
Done!